config/betterquesting/DefaultQuests.json

```
@@ -57800,7 +57709,7 @@
     "betterquesting:10": {
       "pack_name:8": "Enigmatica 2: Expert",
       "hardcore:1": 0,
-      "editmode:1": 0,
+      "editmode:1": 1,
```

```suggestion
      "editmode:1": 0,
```
Better disable edit mode :D

---

Damn it, I knew I forgot something :D